variables.tf

```diff
@@ -172,3 +172,84 @@ variable "report_build_status" {
   default     = false
   description = "Set to true to report the status of a build's start and finish to your source provider. This option is only valid when the source_type is BITBUCKET or GITHUB"
 }
+
+variable "vpc_id" {
+  type    = string
+  default = ""
+}
+
+variable "vpc_subnet_ids" {
+  type    = list
+  default = []
+}
+
+variable "vpc_security_group_ids" {
+  type    = list
+  default = []
+}
+
+variable "git_clone_depth" {
+  type        = number
+  default     = null
+  description = "Truncate git history to this many commits."
+}
+
+variable "private_repository" {
+  type        = bool
+  default     = false
+  description = "Set to true to login into private repository with credentials supplied in source_credential variable."
+}
+
+variable "source_credential_auth_type" {
+  type        = string
+  default     = "PERSONAL_ACCESS_TOKEN"
+  description = "The type of authentication used to connect to a GitHub, GitHub Enterprise, or Bitbucket repository."
+}
+
+variable "source_credential_server_type" {
+  type        = string
+  default     = "GITHUB"
+  description = "The source provider used for this project."
+}
+
+variable "source_credential_token" {
+  type        = string
+  default     = ""
+  description = "For GitHub or GitHub Enterprise, this is the personal access token. For Bitbucket, this is the app password."
+}
+
+variable "source_credential_user_name" {
+  type        = string
+  default     = ""
+  description = "The Bitbucket username when the authType is BASIC_AUTH. This parameter is not valid for other types of source providers or connections."
+}
+
+variable "source_version" {
+  type        = string
+  default     = ""
+  description = "A version of the build input to be built for this project. If not specified, the latest version is used."
+}
+
+variable "fetch_git_submodules" {
```

Comment:
This and a few other variables aren't used... Any reason they around?

Reply:
This will be gone with next push. I think it was a cut and past mistake.

Reply:
Actually I did find the reference to this... looks like I missed a merge... now that I got that sorted I'm finding where I introduced this variable.

It'll be there after my next push.

Reply:
Ugh... hate when I make a merge mistake. Anyway... I believe this has been sorted out now.